### ISSUE TITLE
app rename

### ISSUE THREAD

**Issue by eiliatmr** [iman-salmani/iplan#3]
the app itself looks solid and well made. 
the only suggestion that I have is renaming the app to something less apple-ish.
something else will suit more on linux de's than "iplan" imho.
and it probably make the app more recognizable compared to other alternatives.

'im persian btw :)) damet garm. khaste nabashi"

**Comment by iman-salmani** [comment]
thanks
this name less used. and has funny composition with OurPlan web api.
i will keep this open. if you or any have idea about this feels free to comment.

**Comment by AchillesBoi** [comment]
I have an idea, just rename it to iPlan with a lowecase 'i'.

This way it looks way better than IPlan, and you still get to keep the name.

**Comment by iman-salmani** [comment]
the discussion will be continued on this issue #3.

**Comment by iman-salmani** [comment]
Here is an explanation for people who like to suggest a name.
This app will help users plan their personal life and workflow, which contains tasks, records, notes, and connectivity alongside handy features like integrating with terminal and git.
for connectivity, users can connect to multiple services from Google Tasks to Todoist and Nextcloud. there is also an open-source and privacy-respectful service in the plan which supports all app features.

I have a long list but I think that's enough for this message.

IPlan is the combination of I(user) and Plan(planning). and the name of the service gonna be ourPlan.

**Comment by arvinnmj** [comment]
use 'i' for app just remaining Apple products.
just for the suggestion, use "Plan it" or "planit" .
Planit is combination of plan ( planning ) and it (task).
it's EZ to use (both in speak & type) :)

**Comment by iman-salmani** [comment]
> Planit is combination of plan ( planning ) and it (task).

interesting.
@arvinnmj اسم فارسی به ذهنت می رسه که تو انگلیسی خوب بنظر بیاد؟

**Comment by bertob** [comment]
Hi there, I'd be happy to help brainstorm a new name that fits with [the guidelines](https://teams.pages.gitlab.gnome.org/Design/hig-www/guidelines/app-naming.html). Here are a few ideas to start with:

- Strategist
- Mastermind
- Docket
- Itinerary
- Headquarters
- Paramount

**Comment by AchillesBoi** [comment]
You seem to really like your original "IPlan" name but like I said in my suggestion, changing the "I" for a lowercase "i" is best because:

1) iPlan just looks a lot nicer and cleaner, and you get to keep the original name
2) It avoid confusion on whether the "I" in IPlan is an uppercase "i" or a lowercase "L"

This really is a no-brainer. Chaning it to anything else makes no sense unless you want to move away from the original name.

**Comment by eiliatmr** [comment]
"i"plan is even more apple-ish than Iplan
You are right about it tho but We're trying to find a name that suits gnome naming scheme more

**Comment by AchillesBoi** [comment]
Well, it seems people don't like the idea of "iPlan" simply because it's too Apple-y. I think it looks nice and that's why Apple uses it on its most popular products but none of the new ones have the "i" anymore (macOS, visionOS, Apple Watch, Apple TV, etc.)

I have other suggestions then:

- PlanT
- myPlan
- TaskOn
- myTask
- Plannit
- Taskeroid

**Comment by bertob** [comment]
@AchillesBoi FYI, the [naming guidelines](https://teams.pages.gitlab.gnome.org/Design/hig-www/guidelines/app-naming.html) explicitly advise against made up words, weird capitalization, and the like.

**Comment by AchillesBoi** [comment]
Ah, totally missed the fact that it has to follow GNOME naming guidelines.

How about:

- Puran (japanese for Plan)
- Planbook
- Groundwork
- Devise
- Task King (tasking)
- Plan It
- Task Man

**Comment by bertob** [comment]
Much better! I kinda like Groundwork.

**Comment by iman-salmani** [comment]
> Much better! I kinda like Groundwork.
Yes. Also, I like the names `devise` and `plat it` too. By the way, I should consider available domains.

**Comment by bertob** [comment]
"Plan It" isn't quite in line with our guidelines since it's structured like a sentence, but "Devise" could work.

**Comment by bertob** [comment]
Ping? Plenty of good options here :)

**Comment by iman-salmani** [comment]
Hi @bertob
I think the project should have an organization in GitHub or somewhere else to keep new repositories together And a domain on the web.
So it shouldn't be something already taken.
Also, I Kinda become obsessed with this because I don't want to change its name multiple times.

**Comment by mohsen-w-elsisi** [comment]
I also think the icon could be changed to look more modern, and fit the GNOME style... I'd love to try redisigning the icon, but with discussions of a name change I think the icon should wait until a name is selected, then a complete rebrand would be complete 😃
